includes/install.yaml

```diff
@@ -59,7 +59,7 @@
 
     - name: GitHub Actions Runner | Remove previous configuration before proceeding
       become: true
-      become_user: "{{ gh_runner_service_user }}"
+      become_user: "{{ gh_runner_installer_user }}"
```

Comment:
If we don't do that, in the case where `ansible_user` != `gh_runner_service_user`, we hit the permisison error of ansible_user since the folders are owned by `gh_runner_service_user` and not `ansible_user` (used if no `become` and no `become_user`)